### ISSUE TITLE
Exception calling "ExpandString" -- "Unable to find type [MSBuild]."

### ISSUE THREAD

**Issue by BenKey** [Caphyon/clang-power-tools#550]
I get the following error with my [TTSApp project](https://svn.sullivanandkey.com/SnKOpen/cpp/TTSApp/trunk/).

```
Exception calling "ExpandString" with "1" argument(s): "Unable to find type [MSBuild]."
At C:\users\bkey1\appdata\local\microsoft\visualstudio\15.0_7cc873f3\extensions\r2u1hayh.f3h\psClang\msbuild-expression
-eval.ps1:166 char:9
+         $res = $ExecutionContext.InvokeCommand.ExpandString($expressi ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : RuntimeException

```

**Comment by hero101111** [comment]
Hello,

Thank you for taking the time to report this issue.
It appears that the calls to `[MSBuild]::GetPathOfFileAbove` from your pojrect cause the issue you've experienced.
We will push a fix for this issue as soon as possible and update this thread as soon as progress is made.

Regards,
Gabriel

**Comment by hero101111** [comment]
I'm pleased to inform you that we've pushed the fix for this issue. It will be available in the next release of Clang Power Tools (`v5.0`).

Thank you for your feedback.

**Comment by hero101111** [comment]
Hello again,

A quick update: We've had a bit of rescheduling of the release schedule and version `v4.10` was released a few days ago, instead of `v5.0`. Nonetheless, the fix _is_ included in `v4.10`.

Regards,
Gabriel.
